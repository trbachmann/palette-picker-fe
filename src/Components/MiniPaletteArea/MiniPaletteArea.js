import React, { Component } from 'react';
import MiniPalette from '../../Containers/MiniPalette/MiniPalette';

const MiniPaletteArea = (props) => {
  const palettesToShow = props.projectPalettes.map(palette => {
    return (
      <MiniPalette {...palette} key={palette.id}/>
    )
  })
  return(
    <div>
      {palettesToShow}
    </div>
  )
};

export default MiniPaletteArea;