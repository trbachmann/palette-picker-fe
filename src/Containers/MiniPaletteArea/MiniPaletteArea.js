import React, { Component } from 'react';
import MiniPalette from '../../Components/MiniPalette/MiniPalette';

export default class MiniPaletteArea extends Component{
  render() {
    return(
      <div>
        <MiniPalette/>
      </div>
    )
  }
}