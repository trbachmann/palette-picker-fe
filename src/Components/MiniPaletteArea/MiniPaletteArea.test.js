import React from 'react';
import { shallow } from 'enzyme';
import { MiniPaletteArea } from './MiniPaletteArea';

describe('MiniPaletteArea', () => {
  describe('MiniPaletteArea component', () => {
    let wrapper

    beforeEach(() => {
      wrapper = shallow(<MiniPaletteArea/>);
    });
    
    it('should match the snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});