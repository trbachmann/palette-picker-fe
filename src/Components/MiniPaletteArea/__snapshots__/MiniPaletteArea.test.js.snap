// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MiniPaletteArea MiniPaletteArea component should match the snapshot 1`] = `
<div
  className="MiniPaletteArea--div"
>
  <Connect(MiniPalette)
    color1="#FCB97D"
    color2="#EDD892"
    color3="#C6B89E"
    color4="#B5B8A3"
    color5="#AABA9E"
    created_at="2019-03-19T20:28:06.194Z"
    id={1}
    name="Not favorites"
    project_id={1}
    updated_at="2019-03-19T20:28:06.194Z"
  />
  <Connect(MiniPalette)
    color1="#A0DDFF"
    color2="#758ECD"
    color3="#C1CEFE"
    color4="#7189FF"
    color5="#624CAB"
    created_at="2019-03-19T20:28:06.193Z"
    id={2}
    name="Favorites"
    project_id={1}
    updated_at="2019-03-19T20:28:06.193Z"
  />
</div>
`;
